server/src/main/java/org/opensearch/node/Node.java

```diff
@@ -1047,14 +1048,16 @@ protected Node(
                 admissionControlService
             );
 
+            SearchWorkloadTransportInterceptor searchWorkloadTransportInterceptor = new SearchWorkloadTransportInterceptor(threadPool);
```

Comment:
nit: shall we rename the class to `WorkloadManagementTransportInterceptor` instead of `SearchWorkloadTransportInterceptor`? Or any other suggestions